src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationonsameline/Example1.java

```diff
@@ -13,7 +13,9 @@
 // xdoc section -- start
 class Example1 {
 
-  @SuppressWarnings("deprecation") // violation
+  @SuppressWarnings("deprecation")
+  // violation above, "Annotation 'SuppressWarnings'
+  // should be on the same line with its target."
```

Comment:
please keep comment single-line.

please make it `// violation below ....`
to keep annotation close method and violation marker comment does not make code unrealistic.
Please apply this good practice to other case in your update.

Reply:
Only in ExampleX.java files, such content will be on our website, so it has a bit more strict rules

Reply:
Done, adapted the comment in other classes to keep comments single-lined and attempted to keep the annotation close to method as much as possible. 
I Cut the message in rare cases when the comment was on the same line-
i.e @Ann // violation, "should be..." -- In those cases, I figured adding the name in the comment would not be necessary so, shorten the message in order to satisfy the "close to the method" rule made sense in my opinion. (see InputAnnotationOnSameLineCheckTokensOnMethodAndVar.java 20:3)

Open to other comments